afs-storage-api/src/main/java/com/powsybl/afs/storage/events/ParentChanged.java

```diff
@@ -16,9 +16,17 @@
 
     public static final String TYPENAME = "PARENT_CHANGED";
 
+    @JsonProperty("newParentId")
+    private String newParentId;
+
     @JsonCreator
-    public ParentChanged(@JsonProperty("id") String id) {
+    public ParentChanged(@JsonProperty("id") String id, @JsonProperty("newParentId") String newParentId) {
```

Comment:
From my point of view, you should (if possible) add both `oldParentId` and `newParentId`.